src/core/Akka.API.Tests/CoreAPISpec.ApprovePersistence.approved.txt

```diff
@@ -709,8 +709,9 @@ namespace Akka.Persistence
     }
     public sealed class WriteMessagesFailed : Akka.Actor.INoSerializationVerificationNeeded, Akka.Persistence.IJournalMessage, Akka.Persistence.IJournalResponse, Akka.Persistence.IPersistenceMessage, System.IEquatable<Akka.Persistence.WriteMessagesFailed>
     {
-        public WriteMessagesFailed(System.Exception cause) { }
+        public WriteMessagesFailed(System.Exception cause, int writeCount) { }
```

Comment:
Need to change this to a new overload that gets called from the original constructor with a default value of some kind - otherwise this will break all journal implementations that aren't upgraded to 1.4.11+ right away.

Reply:
I added back the previous constructor and marked it as obsolete --feel free to sugest a better message. `WriteCount` will always be zero in this case, which will make the code behave as before.

Reply:
Sounds like a good solution